testing/integration/src/tester.rs

```diff
@@ -84,6 +90,8 @@ where
             address_map: empty_cid,
             next_id: 100,
             network_name: "test".to_owned(),
+            #[cfg(feature = "m2-native")]
+            installed_actors: empty_list_cid,
```

Comment:
ditto: providing a constructor would let us avoid this.

Reply:
done.